### DISCUSSION TITLE
Filter buttons duplicating

### DISCUSSION THREAD

**Discussion by bum8hj** [community/community#34962] [Mobile] [answered]
Seems to happen only in the **Share Feedback** list under Settings > More Options 

**Version** (if relevant)
GitHub Mobile v1.82.0 (678)

**Video**
https://user-images.githubusercontent.com/27828549/193598208-69efe29d-0ecd-43f1-86a7-6e675ba3d8d9.mp4

**Comment by HenSquared** [comment, marked as answer]
Hi @bum8hj,

Thanks a lot for the video. It made it easy to see how to reproduce this. I'm able to reproduce this on my device as well. I opened an issue with our team to track work on this. Thanks a bunch!

**Comment by aconsuegra** [comment]
@bum8hj Can you share what Android OS version and device are you using? It will help our engineers to track the root cause of the issue, thanks!

**Reply by bum8hj** [reply]
Android 12
Samsung Galaxy S10
